mockrxandroidble/src/main/java/com/polidea/rxandroidble/mockrxandroidble/RxBleConnectionMock.java

```diff
@@ -468,4 +473,9 @@ public Boolean call(byte[] ignored) {
         }
 
     }
+
+    @Override
+    public <T> Observable<T> queue(Func3<Scheduler, BluetoothGatt, RxBleGattCallback, RxBleRadioOperation<T>> builder) {
+        return Observable.never();
```

Comment:
I look at it a little but it seems hard to support it mainly because there is no `BluetoothGatt` nor `RxBleGattCallback` instances available that I could pass to custom operation.

This means we would need to some mock for them (which might be hard for `BluetoothGatt`) or pass `null` which would not exercise the custom operation.

I think people should instead test them the way you guys do in the RxAndroidBLE code base.

As such, I would vote for throwing an exception, probably `UnsupportedOperationException`.